spec/support/generator_spec_helper.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+module Generators
+  extend ActiveSupport::Concern
+
+  ROOT_DIR = File.expand_path('../..', __dir__)
+  SPEC_DIR = File.join(ROOT_DIR, 'spec')
+  TMP_DIR = File.join(ROOT_DIR, 'tmp')
+
+  TEST_PROJECT_NAME = 'rails_test_app'
+
+  SRC_TEST_PROJECT_ROOT_DIR = File.join(SPEC_DIR, TEST_PROJECT_NAME).freeze
+
+  TMP_TEST_PROJECT_ROOT_DIR = File.join(TMP_DIR, TEST_PROJECT_NAME).freeze
+  TMP_TEST_PROJECT_APP_DIR = File.join(TMP_TEST_PROJECT_ROOT_DIR, 'app').freeze
+  TMP_TEST_PROJECT_MODEL_DIR = File.join(TMP_TEST_PROJECT_APP_DIR, 'models').freeze
+  TMP_TEST_PROJECT_MIGRATION_DIR = File.join(TMP_TEST_PROJECT_ROOT_DIR, File.join('db', 'migrate'))
+
+  def generator_class
+    described_class
+  end
+
+  def run_generator(*args, **kw_args)
+    args = Array(args)
+    FileUtils.cd(TMP_TEST_PROJECT_ROOT_DIR) do
+      g = generator_class.new(args, kw_args, {})
+      g.invoke_all
+    end
+  end
+
+  # rubocop:disable Lint/SuppressedException
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary disabling of `Lint/SuppressedException` (unknown cop).](https://app.codacy.com/gh/Zetatango/debugs_bunny/pullRequest?prid=5044072)